postgres/values.yaml

```diff
@@ -18,3 +20,6 @@ hba_cidr: 0.0.0.0/0
 persistence:
   enabled: true
 
+backup:
+  enabled: true
+  image: hub.global.cloud.sap/c5211168/monsoon3-backup:0.1.2
```

Comment:
This needs to be configurable with a value. The default should be a public image `sapcc/backup-tools:v0.1.2`. Internally we overwrite this with out image from hub.global.cloud.sap.